functional-tests/src/test/java/com/redhat/che/functional/tests/RhCheAbstractTestClass.java

```diff
@@ -74,4 +82,12 @@ public void closeFilesAndProject() {
   public void closeBrowser() {
     ide.close();
   }
+
+  public boolean isProd() {
+    return cheHost.equals(CHE_PROD_URL);
+  }
+
+  public boolean isPreview() {
+    return cheHost.equals(CHE_PROD_PREVIEW_URL);
```

Comment:
I would change it to `CHE_PROD_PREVIEW_URL.equals(cheHost)` to avoid NPE

Reply:
Same as my comment above

Reply:
refactored.